overview_tutorials/main_tutorial.ipynb

```diff
@@ -0,0 +1,1991 @@
+    "We will sort the bandpass cached filtered recording the `recording_saved` object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "si.available_sorters()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "si.installed_sorters()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `spikeinterface.sortingcomponents` module includes functions that can be used to create custom spike sorting pipelines built-in in `SpikeInterface`. It is still experimental and under heavy development, but there are already 3  SI-based sorters available:\n",
+    "\n",
+    "* `tridesclous2` (developed by Samuel Garcia)\n",
+    "* `spykingcircus2` (developed by Pierre Yger)\n",
+    "* `simple` which is only for demo and teaching\n",
+    "\n",
+    "They can be run with the same `run_sorter` function, but they don't require any additional installation!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "si.run_sorter?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can retrieve the parameters associated to any sorter with the `get_default_params()` function from the `sorters` module:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "si.get_default_sorter_params('kilosort2_5')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "To modify a parameter, we can easily pass it to the `run` function as an extra argument!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run sorter locally"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# si.Kilosort2_5Sorter.set_kilosort2_5_path('/home/samuel.garcia/Documents/SpikeInterface/code_sorters/Kilosort2.5/')\n",
+    "si.installed_sorters()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorter_params = {'do_correction': False}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "sorting_KS25 = si.run_sorter('kilosort2_5', recording_saved,\n",
+    "                             output_folder=base_folder / 'sorter_KS25',\n",
+    "                             verbose=True, **sorter_params)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can check the output object:"
```

Comment:
An idea: we could include the sorter in the "SpikeInterface Dataset Tutorial" Drive, so that people can run the notebook without having to sort the data themselves?